test/tests/combobox_grid-combo.js

```diff
@@ -877,10 +877,14 @@ ariaTest(
     // Send key "ARROW_HOME"
     await combobox.sendKeys(Key.HOME);
 
-    t.true(
-      await confirmCursorIndex(t, ex.comboboxSelector, 0),
-      'Cursor should be at index 0 after one ARROW_HOME key'
-    );
+    // On macOS, the HOME key deselects the grid popup
+    // but it doesn't move the cursor.
```

Comment:
On macOS, pressing HOME does not move the cursor in the selector input. This was tested on multiple systems manually and with this test. Pressing the key does modify the `aria-activedescendant` as expected but it doesn't have the secondary effect of moving the cursor to the beginning of the input. This likely warrants a TODO or separate follow-up to determine if this should be addressed in the pattern implementation.